### PR TITLE
Use only latest plugin definitions for che-theia and che-machine-exec

### PR DESCRIPTION
**Pull request by ericwill**
Signed-off-by: Eric Williams <ericwill@redhat.com>

### What does this PR do?
Deletes extra definitions and uses `latest` only

### What issues does this PR fix or reference?
Part of CRW-1648

### CODE REVIEW

File: dependencies/che-plugin-registry/che-editors.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2.0.0
 editors:
   - schemaVersion: 2.1.0
     metadata:
-      name: eclipse/che-theia/next
+      name: eclipse/che-theia/latest
```
Comment (nickboldt) [inline dependencies/che-plugin-registry/che-editors.yaml:5]: should line 11 be changed to latest too?

Comment (ericwill) [reply dependencies/che-plugin-registry/che-editors.yaml:5]: you are correct

**Review by nickboldt — APPROVED** [review]
+1

**Comment by benoitf** [comment]
Some PR checks are red

**Comment by ericwill** [comment]
> Some PR checks are red

CI is broken, need to investigate why still

**Comment by benoitf** [comment]
EoF newline ?

**Comment by nickboldt** [comment]
> EoF newline ?

```
[EOF Rule] File "dependencies/LATEST_IMAGES" has lint errors
[EOF Rule] File "dependencies/LATEST_IMAGES_COMMITS" has lint errors
[EOF Rule] File "dependencies/LATEST_IMAGES_DIGESTS.json" has lint errors
```

3 generated files. unrelated to this PR since they were unchanged by this pr. Can I add them to an ignore list? Why did we enable them anyway? what value are these EOF checks providing?

**Comment by benoitf** [comment]
If they're committed they should still have proper EOF ( And if generated it's also easy to generate proper files)

The value is to avoid the diff warning that you have by default when there is no EOF newline

**Pull request merged**